app/models/miq_report/generator.rb

```diff
@@ -93,14 +93,11 @@ def invent_includes
     end
   end
 
-  def include_as_hash(includes = include, klass = nil)
+  def include_as_hash(includes = include, klass = db_class, klass_cols = cols)
     result = {}
-    if klass.nil?
```

Comment:
This exact line got lost. Cannot tell if it is actually issue.

This method is used in recursion. Previously, we entered this blok only on a first go-through. We never entered this in sub-sequent go-throughs. (`klass` was = to `assoc_klass`).

I just spot the difference, no idea if this is a bug or fix. :)